### PR TITLE
colorls: add French translation

### PR DESCRIPTION
**Pull request by Ceres445**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/colorls.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# colorls
+
+> Embellissez la sortie de la commande `ls`, avec des icônes de couleurs et de polices impressionnantes. Disponible sous forme de gemme Ruby.
```
Comment (nicokosi) [inline pages.fr/common/colorls.md:3]: nitpick 🥜 : ‏ Seems better without the comma and "des" , IMHO. :innocent: 
```suggestion
> Embellissez la sortie de la commande `ls` avec des icônes de couleurs et des polices impressionnantes. Disponible sous forme de gemme Ruby.
```

File: pages.fr/common/colorls.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> Plus d'informations: <https://github.com/athityakumar/colorls>.
```
Comment (nicokosi) [inline pages.fr/common/colorls.md:4]:
```suggestion
> Plus d'informations : <https://github.com/athityakumar/colorls>.
```

File: pages.fr/common/colorls.md (hunk 1 above)
Comment (noraj) [inline pages.fr/common/colorls.md:3]:
```suggestion
> Embellit la sortie de la commande `ls`, avec des icônes coloreés ou provenant de font-awesome. Disponible sous forme de gemme Ruby.
```

Third person. font-awesome should be translated as it's a product name.

File: pages.fr/common/colorls.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Lister les fichiers un par ligne :
```
[the remark this answers is not in the document]
Comment (noraj) [reply pages.fr/common/colorls.md:6]: > Command and example descriptions on pages in French must use the third person singular present indicative tense

see

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#french-specific-rules

```suggestion
- Liste les fichiers, un par ligne :
```

File: pages.fr/common/colorls.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Liste tous les fichiers, y compris les fichiers cachés :
+
+`colorls --all`
+
+- Liste au format long (autorisations, propriété, taille et date de modification) de tous les fichiers :
+
+`colorls --long --all`
+
+- Ne lister que les répertoires :
```
Comment (noraj) [inline pages.fr/common/colorls.md:18]:
```suggestion
- Ne liste que les répertoires :
```

**Review by noraj — CHANGES_REQUESTED** [review]
Thanks for your contribution.

**Review by noraj — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Pull request merged**